tools/autodeployment/README.md

```diff
@@ -14,7 +14,7 @@ is created on GitHub).
 There are also a few other routes you can use to get information about the
 auto deployment server:
 
-- `GET :4000/status`: returns JSON data about the current status of the build.
+- `GET /deploy/status`: returns JSON data about the current status of the build.
 - `GET :4000/log`: returns a real-time stream of current build log output, if any.
```

Comment:
What about this one?

Reply:
Oops! Silly mistake. I'm going to change it now.